build/pipelines/fuzz.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+      - docs/*
+      - samples/*
+      - tools/*
+
+pr: none
+
+#     0.0.yyMM.dd##
+#     0.0.1904.0900
+name: 0.0.$(Date:yyMM).$(Date:dd)$(Rev:rr)
+
+stages:
+  - stage: Build_Fuzz_Config
+    displayName: Build Fuzzers
+    dependsOn: []
+    condition: succeeded()
+    jobs:
+    - template: ./templates/build-console-fuzzing.yml
+      parameters:
+        platform: x64
+  - stage: OneFuzz
+    displayName: Submit OneFuzz Job
+    dependsOn: ['Build_Fuzz_Config']
+    condition: succeeded()
+    pool:
+      vmImage: 'ubuntu-latest'
+    variables:
+      artifactName: drop
+    jobs:
+    - job:
+      steps:
+      - task: DownloadBuildArtifacts@0
+        inputs:
+          artifactName: $(artifactName)
+          downloadPath: $(Build.ArtifactStagingDirectory)
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.x'
+          addToPath: true
+          architecture: 'x64'
+      - bash: |
+          set -ex
+          pip -q install onefuzz
+          onefuzz config --endpoint $(endpoint) --client_id $(client_id) --authority $(authority) --tenant_domain $(tenant_domain)
+          sed -i s/INSERT_PAT_HERE/${ado_pat}/ ../Fuzz/notifications-ado.json
+          sed -i s/INSERT_ASSIGNED_HERE/${ado_assigned_to}/ ../Fuzz/notifications-ado.json
+        displayName: Configure OneFuzz
+      # replace the bash command below with this to run this with notifications
+      # onefuzz template libfuzzer basic --colocate_all_tasks --vm_count 1 --target_exe $(target_exe_path) --notification_config ../Fuzz/notifications-ado.json OpenConsole $(test_name) $(Build.SourceVersion) windows
+      - bash: |
+          onefuzz template libfuzzer basic --colocate_all_tasks --vm_count 1 --target_exe $(target_exe_path) OpenConsole $(test_name) $(Build.SourceVersion) windows

```

Comment:
After we've fully reviewed this PR (more specifically notifications.json), I'll do this replacement and enable notifications before we go live.